go/test/endtoend/vtgate/errors_as_warnings/main_test.go

```diff
@@ -136,7 +134,7 @@ func TestScatterErrsAsWarns(t *testing.T) {
 	for _, mode := range modes {
 		t.Run(mode.m, func(t *testing.T) {
 			// connection setup
-			utils.Exec(t, mode.conn, "use @replica")
+			utils.Exec(t, mode.conn, "use ks@replica")
```

Comment:
as we will be skipping the test for partial_keyspace, this change would not be needed.